libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -2877,8 +2895,13 @@ def _construct_responses_api_payload(
     for legacy_token_param in ["max_tokens", "max_completion_tokens"]:
         if legacy_token_param in payload:
             payload["max_output_tokens"] = payload.pop(legacy_token_param)
-    if "reasoning_effort" in payload:
-        payload["reasoning"] = {"effort": payload.pop("reasoning_effort")}
+    if "reasoning_effort" in payload or "reasoning_summary" in payload:
```

Comment:
wonder if it'd be less confusing to have a single `reasoning` param that matches openais